processing/physical_range.py

```diff
@@ -312,93 +310,122 @@ def make_plot(self, x_data, y_data, var_obj, year, thresholds=True):
                      for t in self.data['TIMESTAMP_START'][start:end + 1]]
-        fig = plt.figure(1, figsize=(12, 13))  # Create figure
+        fig, (subplot1, subplot2) = plt.subplots(
+            2, 1, figsize=(12, 13))  # Create figure
         y_name = year.replace("_", " ")
-        fig.suptitle(
-            f'Physical Range of {var_obj.name} throughout {y_name}',
-            fontsize=self.plot_config.plot_suptitle_fontsize)
-
-        self.make_plot(time_data, annual_data, var_obj, year, thresholds=True)
-        self.make_plot(time_data, annual_data, var_obj, year, thresholds=False)
-        """
-        Set up legend with labels and corresponding colors
-        """
-        # Set labels and colors for lines
+
+        text1 = 'Physical Range | '
+        text2 = f'{var_obj.name}'
+        text3 = f' | {y_name}'
+
+        # Calculate the widths of each text segment
+        renderer = fig.canvas.get_renderer()
+        bbox1 = fig.text(0, 0, text1, ha='left', va='top',
+                         fontsize=16).get_window_extent(renderer=renderer)
+        bbox2 = fig.text(0, 0, text2, ha='left', va='top',
+                         fontsize=16, fontweight='bold'
+                         ).get_window_extent(renderer=renderer)
+
+        # Starting x position
+        x_start = 0.01
+
+        # Set the text positions dynamically based on the calculated widths
+        fig.text(x_start, 0.99, text1, ha='left', va='top', fontsize=16)
+        fig.text(x_start + bbox1.width / fig.bbox.width, 0.99,
+                 text2, ha='left', va='top', fontsize=16, fontweight='bold')
+        fig.text(x_start + (bbox1.width + bbox2.width) / fig.bbox.width,
+                 0.99, text3, ha='left', va='top', fontsize=16)
+
+        self.make_plot(subplot1, time_data, annual_data, var_obj,
+                       year, thresholds=True)
+        self.make_plot(subplot2, time_data, annual_data, var_obj,
+                       year, thresholds=False)
         lab_warn = f'Expected Range ({var_obj.min_lim}-{var_obj.max_lim})'
         margin_percent = str(var_obj.margin * 100)
         lab_err = (f'Expected Range +/- {margin_percent}% '
                    f'({var_obj.error_min}-{var_obj.error_max})')
 
-        legend_info = [('orange', lab_warn), ('r', lab_err)]
-        handles = [mlines.Line2D([], [], color=c, label=l) for c, l in
-                   legend_info]  # convert legend info to Patches
-        labels = [l for c, l in legend_info]
+        legend_info = [('#FFAE00', lab_warn), ('#B21B00', lab_err)]
+        handles = [mlines.Line2D([], [], color='None', marker='o',
+                                 markerfacecolor='0.75',
+                                 markeredgecolor='None',
+                                 markersize=10, label='Data')]
+        handles += [mlines.Line2D([], [], color=c, label=l)
+                    for c, l in legend_info]
+        labels = ['Data']
+        labels += [l for c, l in legend_info]

```

Comment:
Parsing of `legend_info` can be modified to do single pass for both `handles` and `labels`